modules/secrets-store-vault.adoc

```diff
@@ -324,7 +324,7 @@ spec:
     spec:
       terminationGracePeriodSeconds: 0
       containers:
-      - image: registry.k8s.io/e2e-test-images/busybox:1.29-4
+      - image: registry.k8s.io/e2e-test-images/busybox:1.30-4
```

Comment:
Might be worthwhile getting QE eyes on this value as it differs slightly than other versions.

Reply:
Yep definitely - I tagged my QE specifically on these ones, here: https://github.com/openshift/openshift-docs/pull/81062#issuecomment-2317618314